syntax = "proto3";
package RPC;

message RPCheader {
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 args_size = 3;
}